device/Microsoft.Azure.Devices.Client.PCL/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Resources;
```

Comment:
This file shouldn't be here...the PCL stuff was removed in master. I'll take care of it...